src/drive/web/modules/actions/useActions.jsx

```diff
@@ -19,13 +18,17 @@ const withHideSelectionBarDispatch = (action, dispatch) => {
 
 const useActions = (actionCreators, actionOptions = {}) => {
   const dispatch = useDispatch()
+  let actions = []
+  actionCreators.map(createAction => {
+    const enhancedAction = withHideSelectionBarDispatch(
+      createAction(actionOptions),
+      dispatch
+    )
+    const name = enhancedAction.icon
```

Comment:
Let's use the "name" attribute then ?

Reply:
there is no name yet. Latter we have `name = label || icon`

Reply:
OK I guess this will be fixed in a next PR then :)